app/js/main.js

```diff
@@ -49,34 +50,34 @@ fileDrop.addEventListener(DRAG_DROP_EVENT, function (e) {
             return;
         }
 
-        const li = document.createElement(LIST_TAG);
-        li.setAttribute(DRAGGABLE_ATTRIBUTE, TRUE_TEXT);
-        li.setAttribute(DATA_FILE_ATTRIBUTE, URL.createObjectURL(file));
-        li.appendChild(document.createTextNode(`${IMAGE_SYMBOL} ${file.name}`));
+        const tr = document.createElement(TABLE_ROW_TAG);
+        tr.setAttribute(DRAGGABLE_ATTRIBUTE, TRUE_TEXT);
+        tr.setAttribute(DATA_FILE_ATTRIBUTE, URL.createObjectURL(file));
+        tr.appendChild(document.createTextNode(`${IMAGE_SYMBOL} ${file.name}`));
```

Comment:
Textnodes within TR are not valid HTML. You need to create a TD tag, add TextNode to it and, put inside TR.

Reply:
Is the table a good idea - in the first place?

Reply:
It may blow up a bit, but, then we simply could extend with image thumbnail or other meta information like dimensions, filesize.

Reply:
So. Let's implement it. Going back to VSC

Reply:
So works now.

Reply:
Works, but still invalid HTML.

put on top to the other consts
`const TABLE_DATA_TAG = 'td';`

```
const td = document.createElement(TABLE_DATA_TAG);
td.appendChild(document.createTextNode(`${IMAGE_SYMBOL} ${file.name}`));
tr.appendChild(td);
```

Reply:
With which tool are you verifying the HTML?

Reply:
No tool, HTML spec: https://de.w3docs.com/html-lernen/html-tag-td.html

Reply:
Finally....